backend/src/main/java/com/empire/Region.java

```diff
@@ -360,6 +360,10 @@ public boolean isSea() {
 		return type == Type.WATER;
 	}
 
+	public boolean hasNoble() {
```

Comment:
This change effectively eliminates the need for Constants.noNoble, if you think you've got rid of all the null/empty string checking for nobles you could axe that variable as well.

Reply:
Done.